dojo/filters.py

```diff
@@ -1538,7 +1539,6 @@ def filter(self, qs, value):
 
 class FindingFilterHelper(FilterSet):
     title = CharFilter(lookup_expr="icontains")
-    date = DateRangeFilter()
```

Comment:
This deletion looks like it is removing the `date` filter for all finding filters besides the report filters. Is this intended?

Reply:
Added this as DateRangeFromToFilter after discussion.